src/components/DatasetCreateView.jsx

```diff
@@ -119,9 +115,17 @@ export class DatasetCreateView extends React.PureComponent {
   }
 
   getStudyOptions(studies) {
-    const studiesList = [];
-    studies.studies.map(study => studiesList.push({ value: study.id, label: study.name }));
-    return studiesList;
+    const studyOptions = [];
+    studies.studies.map(study => studyOptions.push({ value: study.id, label: study.name }));
```

Comment:
Similar comment as below (let's switch to just assigning the result of the map to studyOptions)